src/renderers/noop/ReactNoop.js

```diff
@@ -160,6 +172,14 @@ var ReactNoop = {
     }
   },
 
+  hasScheduledDeferredCallback() {
+    return Boolean(scheduledDeferredCallback);
```

Comment:
I don't think we'll have deferred/animation callbacks in React Native so we might drop this from the API and make that an implementation detail of the React DOM renderer.